packages/remix/src/utils/getIpAddress.ts

```diff
@@ -52,7 +52,7 @@ export function getClientIPAddress(headers: Headers): string | null {
       return parseForwardedHeader(value);
     }
 
-    return value?.split(', ');
+    return value?.split(',').map((v: string) => v.trim());
```

Comment:
Super-l: This _may_ be slightly more efficient as:

```js
return value ? value.replace(/\s*/gm, '').split(',') : null;
```

Reply:
Generally good idea (and I definitely didn't think of this initially) but my only concern here is that we're potentially rewriting individual IP addresses in the header value, _very potentially_ making an invalid IP address appear valid:

`'2b01:cb19:8350:ed00:d0 dd:fa5b:de31:8be5, 141.101.69.35'` would first be rewritten to
`'2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5, 141.101.69.35'` before it's split, resulting in the function returning `2b01:cb19:8350:ed00:d0dd:fa5b:de31:8be5` when it would otherwise have returned `141.101.69.35`. 

Of course, that's a super unlikely edge case but I'd rather avoid it. Wdyt?

Reply:
Wouldn't both code snippets (so the regex and the trim variants) return the same thing in this scenario? 😅  But maybe I am missing something here :) really it probably doesn't matter at all, so let's just go with what you wrote 👍

Reply:
Since `trim` only looks at trailing/leading white spaces it'd not touch the whitespace in the middle of the originally invalid IP address. (I double checked this with a test, just to be sure because I was also confused 😅 ).

But yeah, I'll go with the original version.